www/js/controllers/search-controller.js

```diff
@@ -36,20 +36,13 @@ angular.module('pvta.controllers').controller('SearchController', function($scop
     var stops = [];
     if(StopList.isEmpty()){
       stops = Stops.query(function(){
```

Comment:
Via my comment on the main PR thread, perhaps this should be ``NearestStops.query``?